### PR TITLE
Move routes to own directory

### PR DESCRIPTION
**Pull request by modkaffes**
for better future routes management

This addresses #42

### CODE REVIEW

**Comment by modkaffes** [comment]
Please not this is WIP. I just quickly moved the `subscribe` and `send-email` actions to own files. Will need to fix embedded paths.

**Comment by modkaffes** [comment]
@guylepage3 I checked it locally, works as expected. 

Let me know whether this is enough refactoring or we need more appropriate boilerplating with `/controllers` too.

**Comment by guylepage3** [comment]
Hey @modkaffes, I am just in the middle of a styling refactor. The second I am done, I will pull this in. Thanks.

**Comment by guylepage3** [comment]
Reviewing this now..

File: routes/subscribe/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const subscribe = require('express').Router();
+const request = require('request');
+
```
Comment (guylepage3) [inline routes/subscribe/index.js]: Hey @modkaffes.. It looks as though the `config` settings are not being referenced here. See error below.

Could you make the update to the Pull Request? Thank you sir. :)

```
ReferenceError: config is not defined
    at subscribe.post (/Users/guy/sites/universe.engineering/routes/subscribe/index.js:34:30)
    at Layer.handle [as handle_request] (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/layer.js:95:5)
    at /Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/guy/sites/universe.engineering/node_modules/express/lib/router/index.js:47:12)
```

**Comment by modkaffes** [comment]
> I added the route for config. Great work @modkaffes.. Thank you sir. 👍

Thanks for the fix!

**Pull request merged**